src/build/lrsql/build.clj

```diff
@@ -30,13 +30,19 @@
    {:project "deps.edn"
     :aliases [:db-sqlite :db-postgres]}))
 
+(defn write-git-data! []
+  (when-let [version (b/git-process {:git-args "describe --exact-match --tags"})]
+    (b/write-file {:path "resources/lrsql/config/git-details.edn"
```

Comment:
I believe this should be `target/classes/lrsql/config/git-details.edn`, and you should fire it _after_ the `b/copy-dir` on line 46. That way we don't write a file that has to be git ignored or the like

Reply:
note how the copy draws from the source directory and write s to the classes directory. We can skip the extra step by writing directly to the classes directory after that copy

Reply:
agree, otherwise this needs a .gitignore right away.